engine/script/src/luacjson/lua_cjson.c

```diff
@@ -1558,7 +1560,7 @@ int lua_cjson_decode(lua_State *l, const char* json_string, size_t json_len)
     json_parse_t json;
     json_token_t token;
 
-    json_initialize_config(&cfg);
+    json_initialize_config(&cfg, 0);
```

Comment:
we don't need this buffer for decoding, so it's easier do not allocate it at all